vertica/datadog_checks/vertica/vertica.py

```diff
@@ -626,3 +622,43 @@ def iter_rows(self, view):
 
         for row in cursor.iterate():
             yield row
+
+    def parse_metric_groups(self):
+        default_metric_groups = OrderedDict(
+            (
+                ('licenses', self.query_licenses),
+                ('license_audits', self.query_license_audits),
+                ('system', self.query_system),
+                ('nodes', self.query_nodes),
+                ('projections', self.query_projections),
+                ('projection_storage', self.query_projection_storage),
+                ('storage_containers', self.query_storage_containers),
+                ('host_resources', self.query_host_resources),
+                ('query_metrics', self.query_query_metrics),
+                ('resource_pool_status', self.query_resource_pool_status),
+                ('disk_storage', self.query_disk_storage),
+                ('resource_usage', self.query_resource_usage),
+                ('custom', self.query_custom),
```

Comment:
Should we move `custom` as a metric group? We have custom_query config options and it may be confusing to have to also include them despite enabling options like: https://github.com/DataDog/integrations-core/blob/master/vertica/datadog_checks/vertica/data/conf.yaml.example#L11

If we keep it this way, then should be check that any of the custom_queries options are enabled, we should also automatically include the `custom` metric group to prevent user errors? IMO, i think it might just be safer to always collect custom if they exist (previous logic)

Reply:
Yeah let's remove 👍

Reply:
Thanks, I'll change it back to always include the `custom` metric group!